### PR TITLE
replace recommendation to use default GP

### PR DESCRIPTION
**Pull request by MaratMussabekov**
issue #4370

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

**Comment by mypil** [comment]
@Dansimp - PR has been copy edited and is ready for final review. This is based on issue #4370.

Thank you.

File: windows/security/threat-protection/windows-firewall/isolating-apps-on-your-network.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ All other endpoints that do not meet the previously stated criteria are consider
 
 **To configure a GPO that defines your intranet address space**
 
-1.  Open the Group Policy Management snap-in (gpmc.msc) and edit the Default Domain Policy.
+1.  Open the Group Policy Management snap-in (gpmc.msc), right click on the Group Policy you want to use to define address space, and select **Edit**.
```
Comment (nenonix) [inline windows/security/threat-protection/windows-firewall/isolating-apps-on-your-network.md]:
```suggestion
1.  Open the Group Policy Management snap-in (gpmc.msc), right click on the Group Policy you want to use to define your address space, and select **Edit**.
```

**Comment by MaratMussabekov** [comment]
Dear @nenonix ,
you recommendation was applied, thank you!

**Comment by mypil** [comment]
@Dansimp - PR has been copy edited and is ready for final review. This is based on issue #4370 

Thank you.

**Pull request merged**